src/gameplay/game.ts

```diff
@@ -1322,37 +1338,41 @@ class Game extends Room {
       whoAssassinShot2: this.whoAssassinShot2,
     };
 
-   /*  GameRecord.create(objectToStore, (err) => {
+    GameRecord.create(objectToStore, (err) => {
       if (err) {
         console.log(err);
       } else {
         console.log('Stored game data successfully.');
       }
-    }); */
-    
-    // Now recording games into dailyGameRecord instead of GameRecord. 
-    // The intended workflow is games -> dailyGameRecord -> (at end of rating preiod) Ratings Calculator -> archive into gameRecord
-    DailyGameRecord.create(objectToStore, (err) => {
+    });
+
+    //FR2 - Rating system 1 - record games for rating updates   
+    // This data will be used to calculate ratings at end of rating period.
+    // The intended workflow is games -> gameRatingRecord -> (at end of rating preiod) Ratings Calculator
+    const ratingRecordToStore = {
+      timeGameStarted: this.startGameTime,
+      timeGameFinished: timeFinished,
+      winningTeam: this.winner,
+      spyTeam: this.spyUsernames,
+      resistanceTeam: this.resistanceUsernames,
+      numberOfPlayers: this.playersInGame.length,
+      roomCreationType: this.roomCreationType,
+
+      playerUsernamesOrdered: getUsernamesOfPlayersInGame(this),
```

Comment:
Do we need this? We have spy and resistance team above.

Reply:
I'll remove if it's unnecessary, I figured you might be using that info for something else.

Reply:
Yea nah this table is purely for the rating system, so we should only add stuff relevant to that :)